tools/configure_git.sh

```diff
@@ -1,17 +1,23 @@
-#!/bin/bash
```

Comment:
was that removed by accident?

Reply:
Yes, probably happened when updating the header.